l10n_de_tax_statement/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     'version': '11.0.1.1.1',
     'category': 'Localization',
     'license': 'AGPL-3',
-    'author': 'OpenBIG.org, Onestein, Odoo Community Association (OCA)',
+    'author': 'OpenBIG.org, sewisoft, Onestein, Odoo Community Association (OCA)',
```

Comment:
For one line of code you added, you also added 7 times your name in 5 different files.
By just adding the state to a view, it doesn't seem fair to me that you consider yourself as the author of this module.

Reply:
I'm sorry, I didn't mean to upset you. However, the opposite was already the case, that someboday told me to add my name to the authors section when adding some code. I have removed my company's name from  the authors sections.